### PR TITLE
fix: added csdxConfig

### PR DESCRIPTION
**Pull request by abhinav-from-contentstack**
- added csdxConfig to package.json for analytics
- bumped verson to 1.1.2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "contentstack-cli-content-type",
```
Comment (netrajpatel) [inline package.json:2]: @abhinav-from-contentstack The GH workflow is failing in this PR, can you please fix that?

**Review by netrajpatel — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -67,5 +67,10 @@
     "posttest": "eslint . --ext .ts --config .eslintrc",
     "prepack": "rm -rf lib && tsc -b && oclif manifest && oclif readme",
     "version": "oclif readme && git add README.md"
+  },
+  "csdxConfig": {
+    "shortCommandName": {
+      "cm:stacks:validate-regex": "CTVLD"
```
Comment (netrajpatel) [inline package.json:73]: @abhinav-from-contentstack Is this the valid command `cm:stacks:validate-regex` in the content-type plugin?

Comment (netrajpatel) [reply package.json:73]: @abhinav-from-contentstack This commands seems to be not available in this plugin

File: package.json [hunk 3]
```diff
@@ -67,5 +67,10 @@
     "posttest": "eslint . --ext .ts --config .eslintrc",
     "prepack": "rm -rf lib && tsc -b && oclif manifest && oclif readme",
     "version": "oclif readme && git add README.md"
+  },
+  "csdxConfig": {
+    "shortCommandName": {
```
Comment (netrajpatel) [inline package.json:72]: @abhinav-from-contentstack Can you list down all the commands in this plugin and add their short names?

**Review by netrajpatel — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (abhinav-from-contentstack) [reply package.json]: @netrajpatel I've corrected these command.

**Review by antonyagustine — APPROVED** [review]

**Review by aman19K — APPROVED** [review]

**Pull request merged**